### PR TITLE
Feature/825 fe donation page

### PR DESCRIPTION
**Pull request by nicklee**

### CODE REVIEW

**Comment by nicklee** [comment]
Skinning up the blackbaud donation embed, example here:
https://rca-development.herokuapp.com/pattern-library/render-pattern/patterns/pages/donate/donate.html

File: rca/project_styleguide/templates/patterns/pages/donate/donate.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                        <div class="donate-form" id="bbox-root"></div>
+                    </div>
+                </div>
+            </section>
+            {% if page.further_information %}
+                <section class="section bg bg--dark">
+                <div class="section__row short-course-accordion section__row--last">
+                        {% include "patterns/organisms/accordion-block/accordion-block.html" with accordion_id='1' accordions=page.further_information course_type="short" title=page.further_information_title anchor_heading=True section_id=page.further_information_title|slugify %}
+                </div>
+                </section>
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/pages/donate/donate.html:86]: nitpick: indentation 🧐

File: rca/static_src/sass/components/_donate-form.scss [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+                    background-color: $color--tertiary;
+                    padding-top: 13px;
+                    padding-left: 13px;
+
+                    @include media-query(large) {
+                        width: 80px;
+                        height: 80px;
+                        padding-top: 33px;
+                        padding-left: 28px;
+                        content: url("data:image/svg+xml,%3Csvg width='19' height='19' viewBox='0 0 19 19' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M2.42867 18.4283L15.8166 5.04038L15.8166 11.8286L18.4564 11.8286V0.514899H7.14272V3.15476H13.9309L0.543056 16.5427L2.42867 18.4283Z' fill='white'/%3E%3C/svg%3E%0A");
```
Comment (chris-lawton) [inline rca/static_src/sass/components/_donate-form.scss:103]: You're a css wizard 🧙‍♂️

**Review by chris-lawton — APPROVED** [review]
Top notch

**Pull request merged**